app/layout.tsx

```
@@ -28,7 +106,77 @@ export default function RootLayout({
+            <TextButton>
+              <span className="material-symbols-outlined">play_arrow</span>
+            </TextButton>
+            <TextButton>
+              <span className="material-symbols-outlined">skip_next</span>
+            </TextButton>
+          </div>
+          <div className="song-progress flex items-center gap-4">
+            <p>0:00</p>
+            <div className="song-progress-bar flex-grow bg-[--md-sys-color-on-surface] h-1 rounded-full overflow-clip">
+              <div className="song-progress-bar-inner bg-[--md-sys-color-primary] h-full rounded-full"></div>
+            </div>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)